test/process.jl

```diff
@@ -0,0 +1,7 @@
+# This file is a part of Julia. License is MIT: http://julialang.org/license
+
+@testset "run(``) tests" begin
```

Comment:
better to put these in an existing file (or, in a different PR, split an existing test file apart so it more closely matches the `base/` file names) - any new test file won't actually run unless it's added to `choosetests.jl`